src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,93 @@
+.container {
+  padding-top: 3.5rem;
+  padding-bottom: 2rem;
+  background-color: rgb(29, 40, 48);
+  font-family: Inter, Arial, sans-serif;
+  text-align: left;
+  box-sizing: border-box;
+  font-size: 16px;
+}
```

Comment:
I wonder if you did width: 100% it would stretch to occupy. Might need to do display:block or display:flex too

Reply:
Not sure 😅 . It kind of needs the page layout to be determined first